server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/SetMessages.feature

```diff
@@ -135,16 +135,21 @@ Feature: SetMessages method on shared folders
     Then "bob@domain.tld" ask for message "mDraft"
     And the notFound list should contain "mDraft"
 
-  Scenario: Draft creation in outbox is not allowed
+  Scenario: Draft creation in outbox is allowed
```

Comment:
I don't think so, It should be the only case forbidden

Reply:
Why? 

It do not violate any assumptions, as I might move a $Draft mail to Outbox without resetting $Draft flag.

Why would we care in one case, and not in the other one?

Reply:
>  as I might move a $Draft mail to Outbox without resetting $Draft flag.

Is it allowed ? I thought it wasn't.

Reply:
The entire goal of that ticket is to not require INBOX to reset the $Draft flag while sending a draft ;-)